examples/shader/post_processing.rs

```diff
@@ -147,6 +179,7 @@ fn setup(
             ..Camera2dBundle::default()
         },
         post_processing_pass_layer,
+        UiCameraConfig { show_ui: false },
```

Comment:
My personal take is that this is on the wrong camera. I would assert that UI should be added in the "final" pass without post processing effects applied. Moving this to the other camera also resolves the "wrong ui scale factor" problem, as it will inherit the window's scale factor (rather than assuming a scale factor of 1.0 for the Image rendertarget)

Reply:
But moving this to the other camera only resolves this in the case where the user doesn't want the post processing applied to the UI. What if the user wants the UI to have post processing applied? Seems that if rendering is done at the actual physical size with `window.physical_size()` the UI layout system won't take it into consideration.

Reply:
That is a much less common scenario, so I don't think we should cover it here, especially given the downsides I've already covered. That scenario will be covered by any of the following:
* Implementing post processing in the render graph, after UI is rendered (this is possible now using the new post processing primitives #6415).
* Allowing users to define scale factor overrides for UI for a specific camera (not yet implemented, but it should be).

There is also the approach you employed here as a short term solution, but I don't want to encourage that pattern generally as it encourages the "wrong" outcomes (such as rendering at a resolution lower than you should).

Reply:
We also have plans for a higher level "post processing stack" which will likely subsume all of these scenarios.

Reply:
Okay makes sense, closing the PR then :+1: